src/controllers/user.controller.js

```diff
@@ -267,4 +271,39 @@ export class UserController {
       });
     }
   }
+
+  static async forgotpass(req, res) {
+    try {
+    const user = req.user
+    console.log(user)
+        const resetLink = JWT.sign({ user_email: user.email,
+         user_id: user.id}, 
+          process.env.SECRET_TOKEN, { expiresIn: '10m' });
+          const link = `${process.env.BASE_URL}/reset-password?token=${resetLink}`
+          const resetMessage = resetPasswordTemplate(user.email,link)
+          sendEmail( 
+            emailConfig({
+            email: user.email,
+            subject: "reset password",
+            content: resetMessage,
+          })
+          );
+        return res.status(200).json({ message: link} );
+      } catch(error) {
+      res.status(500).json({ error: error.message });
+    }
```

Comment:
- Remove all logs
- Use the implement util function to generate a token
- Catch block is missing a message, Ask Yves
- Rename your controller to follow camelCase convention